packages/app/src/app/components/CreateNewSandbox/CreateSandbox/CreateSandbox.tsx

```diff
@@ -60,162 +70,221 @@ export const CreateSandbox: React.FC<CreateSandboxProps> = ({
   const essentialState = useEssentialTemplates();
 
   const isSyncedSandboxesPage = location.pathname.includes('/synced-sandboxes');
-  const isDashboardPage = location.pathname.includes('/dashboard');
   const isUser = user?.username === activeTeamInfo?.name;
 
+  /**
+   * Checking for user because user is undefined when landing on /s/, even though
+   * hasLogIn is true.
+   */
+  const showTeamTemplates = hasLogIn && user;
+
   const tab = useTabState({
     orientation: 'vertical',
-    selectedId: initialTab || isSyncedSandboxesPage ? 'import' : 'create',
+    selectedId: initialTab || isSyncedSandboxesPage ? 'import' : 'quickstart',
   });
 
-  const dashboardButtonAttrs = isDashboardPage
-    ? {
-        onClick: actions.modals.newSandboxModal.close,
-      }
-    : {
-        to: '/dashboard/recent',
-        onClick: actions.modals.newSandboxModal.close,
-      };
+  const [viewState, setViewState] = useState<
+    'initial' | 'fromTemplate' | 'fork' /* | 'search' */
+  >('initial');
+  // ❗️ We could combine viewState with selectedtemplate to limit the amout of states.
+  const [selectedTemplate, setSelectedTemplate] = useState<TemplateFragment>();
```

Comment:
WDYT? This can also be combined with the imported repo data and search if necessary. One state reducer to rule them all.